src/einsteinpy/symbolic/tensor.py

```diff
@@ -307,3 +307,11 @@ def symbols(self):
         
         """
         return self.syms
+
+    def function(self):
+        """
+        Returns arrays of numpy like functions rather than sympy expressions
+
+        """
+        numeric_arr = sympy.lambdify((), self.arr, np)
```

Comment:
I told you to drop the `*args` because the variables are already defined in `syms` and `variables`.

Anyways, change it to `function(self, *args)`, and change do a slight edit
```
if len(args)==0:
    numeric_arr = sympy.lambdify([*self.syms, *self.variables], self.arr, np)
else:
    numeric_arr = sympy.lambdify(args, self.arr, np)
```

Try to understand what `self.variables` contains!

Reply:
Got it! Should I make the test?